api/src/main/java/com/mybooks/api/model/Author.java

```diff
@@ -0,0 +1,29 @@
+package com.mybooks.api.model;
+
+import com.sun.istack.NotNull;
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
You can use `@Data` https://projectlombok.org/features/Data

Reply:
I got a warning that using Data may affect JPA performance, wdyt?

Reply:
That's correct, please forget about it.